packages/testlab/src/test-sandbox.ts

```diff
@@ -151,7 +151,7 @@ export class TestSandbox {
       await outputFile(dest, content, {encoding: 'utf-8'});
     }
 
-    if (parse(src).ext === '.js' && pathExists(src + '.map')) {
+    if (parse(src).ext === '.js' && (await pathExists(src + '.map'))) {
```

Comment:
`pathExists` returns a `Promise`, therefore this condition was always resolved as truthy.